doc/getting-started.md

```diff
@@ -147,6 +147,62 @@ $ curl -X 'GET' \
 {"_id":"615f30020eb7c3c6616e5ac3","username":"bob","hashed_password":"$2b$12$VtfVij6zz20F/Qr0Ri18O.11.0LJMMXyJxAJAHQbKU0jC96eo2fr.","active":true}
 ```
 
+### Setup SSH keys
+
+We need to generate SSH keys to use SSH docker container.
```

Comment:
It's probably worth adding a line about the fact that this container is used for storage (even though it wasn't mentioned on the README, just some minor improvement we could make now) and that the files are then available for download via HTTP from the Nginx container.

Reply:
Updated.